pkg/events/events.go

```diff
@@ -169,6 +175,12 @@ func (e *Processor) subscribe(ctx context.Context) error {
 			if err := e.state.Set(block.Number); err != nil {
 				return errors.Wrap(err, "failed to commit last block number")
 			}
+		case newSub := <-e.update:
+			// if new update is received then the connection is broken as noded only issues new update only if
+			// the old manager is broken so we need to resubscribe with the new manager.
+			e.sub = newSub
+			log.Info().Msg("new substrate manager received, trying to resubscribe to the new manager")
```

Comment:
that is not correct u r printing log that there is a new substrate manager that u will try 
then after that directly u print an error which will confuse the user for sure 
the error should be printed first then the info for trying new connection should come after when u r actually trying

Reply:
will remove the log.Info message, will just log that the subscribtion failed and will try again, noded module will log that it was updated successfully once the update is done

Reply:
also both noded and api-gateway modules logs the new urls that will be used, and the error messages of the modules trying to reach the chain shows the faulty urls